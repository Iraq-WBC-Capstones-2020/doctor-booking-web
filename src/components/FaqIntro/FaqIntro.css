img {
  width: 100%;
}
/*# sourceMappingURL=FaqIntro.css.map */
