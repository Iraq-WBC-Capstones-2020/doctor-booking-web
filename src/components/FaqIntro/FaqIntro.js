import React from 'react';
import { Row, Col, Container } from 'react-bootstrap';
import intro from './intro.svg';

function FaqIntro() {
  return (
    <Container>
      <Row className="my-5">
        <Col lg="6" className=" d-flex align-items-center">
          <div>
            <h1 className="my-3">
              Frequently asked<span className="d-block">questions</span>
            </h1>
            <p>
              Lorem ipsum dolor sit amet consectetur adipisicing elit.
              Perspiciatis officiis recusandae voluptatibus! Natus voluptatum
              esse temporibus saepe quo omnis alias!
            </p>
          </div>
        </Col>
        <Col lg="6" className="">
          <img src={intro} className="img-fluid IntroSection__image" alt="" />
        </Col>
      </Row>
    </Container>
  );
}

export default FaqIntro;
