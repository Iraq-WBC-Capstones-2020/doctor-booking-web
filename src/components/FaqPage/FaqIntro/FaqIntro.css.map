{
    "version": 3,
    "mappings": "AAAA,AAAA,QAAQ,CAAC;EACP,cAAc,EAAE,GAAG;EACnB,WAAW,EAAE,IAAI;EACjB,SAAS,EAAE,IAAI;EACf,KAAK,EAAE,OAAO;CACf",
    "sources": [
        "FaqIntro.scss"
    ],
    "names": [],
    "file": "FaqIntro.css"
}