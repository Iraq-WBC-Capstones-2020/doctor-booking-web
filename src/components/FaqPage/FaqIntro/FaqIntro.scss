img {
  width: 100%;
}

.faqText {
  letter-spacing: 1px;
  line-height: 28px;
  font-size: 16px;
  color: #7d7987;
}
