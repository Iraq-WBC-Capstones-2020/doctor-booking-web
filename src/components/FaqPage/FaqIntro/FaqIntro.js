import React from 'react';
import { Row, Col, Container } from 'react-bootstrap';
import intro from './intro.svg';
import './FaqIntro.css';
import { useTranslation } from 'react-i18next';

function FaqIntro() {
  const { t } = useTranslation('validation');
  return (
    <Container>
      <Row className="my-5">
        <Col lg="6" className=" d-flex align-items-center">
          <div>
            <h1 className="my-3">
              {t('validation:FAQ.headLine')}
              <span className="d-block">{t('validation:FAQ.headLine2')}</span>
            </h1>
            <p className="faqText">{t('validation:FAQ.paragraph')} </p>
          </div>
        </Col>
        <Col lg="6" className="">
          <img src={intro} className="img-fluid introSectionImage" alt="" />
        </Col>
      </Row>
    </Container>
  );
}

export default FaqIntro;
