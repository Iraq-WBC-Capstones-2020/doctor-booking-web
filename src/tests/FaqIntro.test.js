import React from 'react';
import FaqIntro from '../components/FaqIntro/FaqIntro';
import renderer from 'react-test-renderer';

test('intro section of faq page render correctly', () => {
  const tree = renderer.create(<FaqIntro />).toJSON();
  expect(tree).toMatchSnapshot();
});
