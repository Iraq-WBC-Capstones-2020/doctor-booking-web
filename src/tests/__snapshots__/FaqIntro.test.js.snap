// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intro section of faq page render correctly 1`] = `
<div
  className="container"
>
  <div
    className="my-5 row"
  >
    <div
      className=" d-flex align-items-center col-lg-6"
    >
      <div>
        <h1
          className="my-3"
        >
          Frequently Asked
          <span
            className="d-block"
          >
            Questions
          </span>
        </h1>
        <p
          className="faqText"
        >
          Here is some frequently asked questions 
          <br />
           about the website, it will help you to get more 
          <br />
           info about the website 
           
        </p>
      </div>
    </div>
    <div
      className="col-lg-6"
    >
      <img
        alt=""
        className="img-fluid introSectionImage"
        src="intro.svg"
      />
    </div>
  </div>
</div>
`;
